src/nplinker/metabolomics/gnps/gnps_format.py

```diff
@@ -5,9 +5,10 @@
 import zipfile
 from bs4 import BeautifulSoup
 from bs4 import Tag
-import requests
+import httpx
```

Comment:
They do almost the same things but both are dependencies of NPLinker. Keeping only one and sticking to it will make life easier ;-)   
Why keeping `httpx` then? It's kind of `requests+`, having all features of `requests` and more. One of the important new features is asynchronous request, which helps scale the downloading. Probably we'll use it later.